src/adapters/postgres/fields-adapter.ts

```
@@ -362,33 +370,33 @@ export class PostgresFieldsService implements IServicelocatorfields {
+            (!sourceFieldName['controllingfieldfk'] ||
+              sourceFieldName['controllingfieldfk'] === '')
           ) {
-            storeWithoutControllingField.push(sourceFieldName["name"]);
+            storeWithoutControllingField.push(sourceFieldName['name']);
           }
 
           const query = `SELECT "name", "value" 
           FROM public.${fieldsData.sourceDetails.table} 
-          WHERE value = '${sourceFieldName["value"]}' 
-          GROUP BY  "name", "value"`
+          WHERE value = '${sourceFieldName['value']}' 
```

ESLint

[error] Replace `'value'` with `"value"`

(prettier/prettier)